### DISCUSSION TITLE
React+Pocketbase realtime tutorial for beginners.

### DISCUSSION THREAD

**Discussion by rajesh6161** [pocketbase/pocketbase#745] [Show and tell]
I have made a simple tutorial using React+PocketBase that can help other new developers to integrate PocketBase into their projects.
It's a real-time todo app that is simple and easily understandable.
Here's the link: https://dev.to/rajesh6161/realtime-todo-app-using-react-and-pocketbase-3mf
GitHub link for the project: https://github.com/rajesh6161/pocketbaseTodo

![image](https://user-images.githubusercontent.com/40054161/194719818-c28f3535-fff1-43fa-b30e-1504322c4a20.png)

**Comment by ganigeorgiev** [comment]
Thank you. I'll move the issue to "Show and tell" discussion.

**Comment by benallfree** [comment]
Hello @rajesh6161, great job! I've added this to the Showcase section of [awesome-pocketbase](https://github.com/benallfree/awesome-pocketbase/)

**Comment by rajesh6161** [comment]
@benallfree thanks a lot.

**Comment by pragmatech** [comment]
whereas this one seems to be acting just as expected! (and with the same db version as the dysfunctional blog one) 
thanks

**Reply by rajesh6161** [reply]
Hey @pragmatech thanks for trying it out! I saw your comment on previous blog app, you can create an issue and describe what you are facing so that we can have proper idea to deal with it.

**Reply by pragmatech** [reply]
thanks for your reply. I'm sort of challenged in dev-speak via github but will try and embrace more terse clear comments heh. Dunno why, but in my version of rabbit-hole-diving that is web dev – and in wanting to build nice simple 'little bicycles' (metaphor for web apps shorn of the behemouth deps, AWS, bloat, hype etc) this way is very attractive. Fwiw, it would be another delight to see how a solid 'User Presence' functionality could work within PB's abilities. (Ok. NEXT time I'll comment tersely heh) thanks
